src/movies.js

```diff
@@ -1,15 +1,58 @@
+
+
 // Iteration 1: All directors? - Get the array of all directors.
+const getAllDirectors = movies.map(function(movie){
+    return(movie.director)
+  })
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-
+function howManyMovies(movies) {
+    const dramaMovies = movies.filter(function (movie) {
+      if (movie.genre.includes("Drama") === true && movie.director === "Steven Spielberg") {
+        return true;
```

Comment:
Instead of returning inside the if statement, the condition itself can be returned.